x-pack/plugins/ml/public/components/job_select_list/job_select_button.html

```diff
@@ -1,14 +1,23 @@
 <div style='display: flex;' class="ml-job-select-btn-container">
-  <div class="ml-job-select-btn-label">Job</div>
+  <div
+    class="ml-job-select-btn-label"
+    i18n-id="xpack.ml.jobSelectButton.jobTitle"
+    i18n-default-message="Job"
+  ></div>
   <div style="flex:0 0 auto;">
     <div  popover-placement="bottom"
           popover-placement="bottom"
           popover-html-unsafe="{{unsafeHtml}}"
           popover-append-to-body="false"
           popover-trigger="click"
           class="ml-job-select-btn"
-          tooltip="{{description.txt}} selected"
-          aria-label="Job selection menu"
+          tooltip="{{:: 'xpack.ml.jobSelectButton.jobSelectionMenuButtonTooltip' | i18n: {
+            defaultMessage: '{description} selected',
+            values: {
+              description: description.txt,
```

Comment:
The tooltip doesn't correctly pick up `description.txt`. In the screenshots below, you can see that `description.txt` gets correctly rendered inside the dropdown's element via `<span class="ml-job-select-btn-text">{{description.txt}}</span>`, but the tooltip is only showing ` šéļļéçţţéð`, it should be `çŵ-ññéţŵŵôŕķķ-15ɱ-1823 àññð 6 6 ôţţĥéŕŕš  šéļļéçţţéð`.

![image](https://user-images.githubusercontent.com/230104/50637670-33fa6800-0f5b-11e9-96ed-086f185d7fdf.png)

![image](https://user-images.githubusercontent.com/230104/50637689-45437480-0f5b-11e9-9663-5666bcb97b75.png)